service/tasks/volume.py

```diff
@@ -154,62 +154,49 @@ def mount_volume_task(driverCls, provider, identity, instance_id, volume_id,
 # Libcloud Instance Action (Attachment) tasks
 
 
-@task(name="attach_task",
-      default_retry_delay=20,
-      ignore_result=False,
-      max_retries=1)
```

Comment:
ignore_result's default is `False`, we don't need to retry so retry delay and max_retries have been removed